ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/point/DropAPinTaskViewModel.kt

```diff
@@ -32,9 +36,15 @@ class DropAPinTaskViewModel
 constructor(resources: Resources, private val uuidGenerator: OfflineUuidGenerator) :
   AbstractTaskViewModel(resources) {
 
+  private var pinColor: Int = 0
```

Comment:
nit: Should we make it `lateinit var` instead of assigning a default?

Reply:
Unfortunately 'lateinit' modifier is not allowed on properties of primitive types